lib/rspec/core/configuration.rb

```diff
@@ -1412,13 +1415,32 @@ def self.delegate_to_ordering_manager(*methods)
       #     end
       #   end
       #
-      #   describe MyClass, :order => :reverse do
+      #   RSpec.describe 'MyClass', :order => :reverse do
       #     # ...
       #   end
       #
       # @note Pass the symbol `:global` to set the ordering strategy that
       #   will be used to order the top-level example groups and any example
       #   groups that do not have declared `:order` metadata.
+      #
+      # @example
+      #   RSpec.configure do |rspec|
+      #     rspec.register_ordering :global do |examples|
+      #       acceptance, other = examples.partition do |example|
+      #         example.metadata[:type] == :acceptance
```

Comment:
I'd actually use 3 types: `:unit`, `:acceptance` and everything else. (The order would be: [`:unit`, everything else, `:acceptance`]).

Makes the order more explicit, it's also more practical and easier to adapt to real life scenarios. (And without losing on readability).

Ordering is a bit hard to "test" if you're a user, so the less tweaking/adapting an example needs in real life, the better.

It's also possible to botch something and accidentally "filter out" a chunk of the unit tests.